src/implementations/twig/components/checkbox/checkbox.story.js

```diff
@@ -14,35 +7,137 @@ import dataDefault from '@ecl/specs-component-checkbox/demo/data--default';
 import checkboxGroup from './checkbox-group.html.twig';
 import notes from './README.md';
 
-const prepareCheckbox = (data) => {
-  getFormGroupKnobs(data);
+const getArgTypes = (data) => {
+  const argTypes = {};
+  argTypes.label = {
+    name: 'legend',
+    type: { name: 'string', required: true },
+    defaultValue: data.label,
+    description: 'The heading of the checkbox group',
+    table: {
+      type: { summary: 'string' },
+      defaultValue: { summary: '' },
+      category: 'Content',
+    },
+    control: {
+      type: 'text',
+    },
+  };
+  argTypes.helper_text = {
+    name: 'group helper text',
+    type: { name: 'string' },
+    defaultValue: data.helper_text,
+    description: 'Helper information to describe the checkbox button group',
+    table: {
+      type: { summary: 'string' },
+      defaultValue: { summary: '' },
+      category: 'Content',
+    },
+    control: {
+      type: 'text',
+    },
+  };
+  argTypes.invalid_text = {
+    name: 'error message',
+    type: { name: 'string' },
+    defaultValue: data.invalid_text,
+    description:
+      'A message informing the user if there is a problem with their entry',
+    table: {
+      type: { summary: 'string' },
+      defaultValue: { summary: '' },
+      category: 'Content',
+    },
+    control: {
+      type: 'text',
+    },
+  };
+  argTypes.optional_text = {
+    name: 'optional text',
```

Comment:
In cases like this, where the argType property is equal to its name you can omit the last one.

Reply:
Interesting but the underscore remain, I'll therefore leave it like this.